src/Controller/Organization/MembersController.php

```diff
@@ -52,12 +55,20 @@ public function listMembers(Organization $organization, Request $request): Respo
     }
 
     /**
-     * @Route("/user/invitation/{token}", name="organization_accept_invitation", methods={"GET"}, requirements={"token"="%uuid_pattern%"})
```

Comment:
instead of changing the path (b.c. break), I would prefer to add an entry to `security.yaml`:
```
- { path: ^/user/invitation., roles: [ROLE_USER, IS_AUTHENTICATED_ANONYMOUSLY] }
```

Reply:
absolutly right, i have reverted and added the security access control entry